eng/pipelines/release-cli.yml

```diff
@@ -404,7 +405,7 @@ stages:
               pwsh: true
               targetType: filePath
               filePath: cli/installer/windows/test-win-msi.ps1
-              arguments: $(AdditionalTestParameters)
+              arguments: -PerMachine:$$(PerMachine)
```

Comment:
Was the double dollar intentional? Not familiar with that syntax.

Reply:
This syntax comes from the interplay between PowerShell and DevOps' macro syntax.

It handles encoding for the cmdlet switch. The `$(PerMachine)` is rewritten as the literal string `true` or `false` and the expression becomes `-PerMachine:$false` and PowerShell can parse. 

If we used a single dollar sign (i.e. `-PerMachine:$(PerMachine)`) the result would translate to `-PerMachine:false` and you'd get an argument error for passing a string to `-PerMachine` instead of a `SwitchParameter` 

We could also use `1` or `0` in the matrix value and avoid double dollar sign.